mrdo_arcade.asm

```diff
@@ -2345,11 +2345,22 @@ DEAL_WITH_BALL:
 	AND		7FH
 	OR		40H
 	LD		(IY+0), A
+
+	LD      A, ($72C3)         ; Load chomper mode flag
+	AND     0FEH               ; Mask off lowest bit (FE = 11111110)
+	CP      80H                ; Check if it's 0x80 or 0x81

```

Comment:
if you want to test bit 7, the correct sequence is:

LD A, ($72C3) ; Load chomper mode flag
AND $80        ; now we have 0 or 128 (i.e. bit 7)
JR    Z,.continue_normally ;  if Z the bit 7 is reset and we are i normal mode